src/app/catalogue/Catalogue.java

```diff
@@ -72,23 +72,23 @@ private static void printSearchResults(int count, String output) {
      * This method allows the user to search for an Item by its Title and return a list of the items that meet the
      * criteria.
      *
-     * @param list        This parameter is the list that the criteria needs to search through.
-     * @param titleSearch This is the input criteria the uses to search through the list of items.
+     * @param list         This parameter is the list that the criteria needs to search through.
+     * @param searchString This is the input criteria the uses to search through the list of items.
      * @return List This method returns a new list of all the Item objects that met the search criteria.
      */
-    public static List<Item> searchByTitle(List<Item> list, String titleSearch) {
+    public static List<Item> searchByTitle(List<Item> list, String searchString) {
```

Comment:
if the searchString equals to an empty string then the method should be exited and an appropriate message should be displayed.